required/latex-lab/documentmetadata-support.dtx

```diff
@@ -137,37 +137,39 @@
 %      The |u| variants do not enforce unicode,
 %      but they will pass the information to hyperref. The |a| variants
 %      do \emph{not} enforce (or even test) a tagged pdf yet.
-%      
-%      Starting with version 0.95s of \pkg{pdfmanagement-testphase}
-%      it is also possible to use the values
-%      |X-4|, |X-4p|, |X-5g|, |X-5n|, |X-5pg|, |X-6|, |X-6n|, |X-6p|, |UA-1| for
-%      a PDF/X and PDF/UA standard. These keys currently set \emph{only} the relevant
-%      XMP-metadata. In version 0.95z support for |UA-2| has been added but
-%      note that |UA-2| hasn't been released yet. It should be used only together
-%      with pdf version 2.0.
+%
+%      Beside the A-standards it is also possible to use the values
+%      |X-4|, |X-4p|, |X-5g|, |X-5n|, |X-5pg|, |X-6|, |X-6n|, |X-6p| for
+%      a PDF/X and |UA-1|  and |UA-2| for PDF/UA standard.
+%      |UA-2| should only be used together with PDF 2.0.
+%      These keys currently set mostly \emph{only} the relevant
+%      XMP-metadata.
```

Comment:
a) not really a sentence I think
b) not sure what this means with respect to UA-1/2

Reply:
a) not sure how to say it differently b) it means e.g. that setting UA-1/2 doesn't automatically activate tagging or load testphase modules. All the standard keys declare the intended standard in the XMP, sometimes they trigger warnings, sometimes they actually do something (like adding a colorprofile), but only few requirements are really enforced and you have to use a validator to check.

Reply:
a)

I think many people would expect that setting, say UA-2 actually tries to prpduce a UA-2 file not just "claims" that the file is one, so I think this needs to be spelled out. But the fact that they are sometime to more, seems to me a bit of a interface issue to be looked at at some point

b) something like this?

These keys declare the intended standard in the XMP-metadata, but do not initiate other actions such as tagging (which is currently activated through the testphase key) to provide the conformance that is claimed by setting the value. In some cases they trigger warnings if the combinations of settings are obviously wrong and in a few cases they doe some extra work in the background (by adding a necessary color profile, for example) but on the whole you need a validator to check that the resulting PDF actually conforms to the claims that are made with the specified settings.

Reply:
well that is already there, in the paragraph above.

Reply:
that wasn't visible in the diff

Reply:
What does the "only" mean here?
The sentence needs revision!

Reply:
I see this has been fixed by now.  Good!